docker/openchallenges/services/mcp-server-standalone.yml

```diff
@@ -1,14 +1,22 @@
+# ------------------------------------------------------------------------------
+# OpenChallenges MCP Server - Docker Compose Configuration
+#
+# This file runs the OpenChallenges Model Context Protocol (MCP) server.
+# It is intended for running the MCP server standalone for local development or testing.
+#
+# To start the server:
+#   docker compose -f path/to/mcp-server-standalone.yml up
+#
+# By default, the server listens on port 8887.
+# ------------------------------------------------------------------------------
+
 services:
   openchallenges-mcp-server:
-    image: ghcr.io/sage-bionetworks/openchallenges-mcp-server:latest
-    container_name: openchallenges-mcp-server-standalone
+    image: ghcr.io/sage-bionetworks/openchallenges-mcp-server:sha-b5013cc
+    container_name: openchallenges-mcp-server
     restart: always
     ports:
-      - '8887:8887'
+      - '${OPENCHALLENGES_MCP_SERVER_PORT:-8887}:8887'
```

Comment:
That should not work. Can you see why? Keep it simple and just use `- '8887:8887'`